meta-resin-common/recipes-core/util-linux/util-linux_%.bbappend

```diff
@@ -1,2 +1,6 @@
+FILESEXTRAPATHS_append := ":${THISDIR}/files"
+
+SRC_URI += "file://4360f56cf6e40b10ab6ace4cf74073de2381c192.patch"
```

Comment:
Maybe rename the patch to 0001-libblkid-don-t-check-for-size-on-UBI-char-dev.patch?